### PR TITLE
Adds the states in Germany

### PR DESCRIPTION
**Pull request by ntodorova**
The module adds states in Germany.

### CODE REVIEW

File: l10n_de_country_states/__init__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from res_country import *
```
Comment (OSguard) [inline l10n_de_country_states/__init__.py]: should be:
from . import res_country

**Comment by OSguard** [comment]
@copmn can you make a review here

**Comment by OSguard** [comment]
:+1:

**Comment by mncoon** [comment]
:+1:

**Pull request merged**